plugins/rollbar/test/samson_rollbar/rollbar_user_informer_test.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require_relative '../test_helper'
+
+SingleCov.covered! uncovered: 1
+
+describe SamsonRollbar::RollbarUserInformer do
+  let(:env) { { 'rollbar.exception_uuid' => 1234 } }
+  let(:body) { ['Original <!-- ERROR NOTIFIER ERROR --> Body', 'continues'] }
+  let(:response) { [200, {}, body] }
+  let(:app) { SamsonRollbar::RollbarUserInformer.new(->(_env) { response }) }
+  let(:call) { app.call(env) }
+  let(:expected_body_with_replacement) do
+    <<~HTML.chomp
+      Original <br/><br/>
+      <a href=https://rollbar.com/instance/uuid?uuid=1234>View error 1234 on Rollbar</a>
+       Body
+    HTML
+  end
+
+  before { SamsonRollbar::RollbarUserInformer.user_information_placeholder = '<!-- ERROR NOTIFIER ERROR -->' }
+
+  it 'informs the user of the error' do
+    call.must_equal(
+      [
+        200,
+        {"Content-Length" => "117", "Error-Id" => "1234"},
+        [expected_body_with_replacement, "continues"]
+      ]
+    )
+  end
+
+  it 'does nothing when user information placeholder is not set' do
+    SamsonRollbar::RollbarUserInformer.user_information_placeholder = nil
+    SamsonRollbar::RollbarUserInformer.expects(:replacement).never
+
+    call.must_equal([200, {}, ["Original <!-- ERROR NOTIFIER ERROR --> Body", "continues"]])
+  end
+
+  it 'does nothing when the error uuid is not given' do
+    SamsonRollbar::RollbarUserInformer.expects(:replacement).never
+
+    app.call({}).must_equal([200, {}, ["Original <!-- ERROR NOTIFIER ERROR --> Body", "continues"]])
+  end
+
+  it 'can handle empty body' do
+    body.clear
+    call.must_equal([200, { "Content-Length" => "0", "Error-Id" => "1234" }, []])
+  end
+
+  describe "when body is an IO" do
+    class FakeIoBody
```

Comment:
this is a global class now ... so better namespace it ... otherwise use `Class.new do`

Reply:
... maybe use StringIO